apps/settings/sub_menu/about_controller.cpp

```diff
@@ -112,11 +112,22 @@ bool AboutController::handleEvent(Ion::Events::Event event) {
       if(childLabel == I18n::Message::Battery){
         MessageTableCellWithBuffer * myCell = (MessageTableCellWithBuffer *)m_selectableTableView.selectedCell();
         char batteryLevel[5];
-        if(strchr(myCell->accessoryText(), '%') == NULL){
-          int batteryLen = Poincare::Integer((int) ((Ion::Battery::voltage() - 3.6) * 166)).serialize(batteryLevel, 5);
-          batteryLevel[batteryLen] = '%';
-          batteryLevel[batteryLen+1] = '\0';
-        }else{
+        if(strchr(myCell->accessoryText(), '%') == NULL) {
+
+            float voltage = (Ion::Battery::voltage() - 3.6) * 166;
+            if(voltage < 0.0) {
+            myCell->setAccessoryText("0%"); // We cheat... => I don't agree : 0% is 0% not 1% x)
```

Comment:
It is very strange for the user to have 0% battery but a calculator still on. And smartphones also do this (1% can stay on for a long time).

Reply:
```suggestion
            myCell->setAccessoryText("1%"); // We cheat...
```